src/eslint/configuration.js

```diff
@@ -158,29 +158,35 @@ const getTypescriptOverride = ({ rootDir }) => {
       tsconfigRootDir: rootDir,
       extraFileExtensions: [".cjs"],
     },
-    files: "{*,**,**/*}.ts",
+    files: "{*,**,**/*}.{ts,tsx}",
     rules: { ...baseRules, ...typescriptRules },
   };
 };
 
 const getJestTypescriptOverride = () => {
   return {
-    files: "{*,**,**/*}.spec.ts",
+    files: ["{*,**,**/*}.spec.ts", "**/__mocks__/{*,**,**/*}.{js,cjs,mjs,ts}"],
     plugins: ["jest"],
     rules: { "@typescript-eslint/unbound-method": "off", "jest/unbound-method": "error" },
   };
 };
 
-const getConfiguration = ({ typescript } = {}) => {
+const getConfiguration = ({ typescript, browser } = {}) => {
   const overrides = [];
 
   if (typescript) {
     overrides.push(getTypescriptOverride(typescript));
     overrides.push(getJestTypescriptOverride());
   }
 
+  const env = { node: true };
+
+  if (browser) {
+    env.browser = true;
```

Comment:
node remains to be true here?

Reply:
Yeah, at least for now, we don't have any project that wouldn't require node environment.